src/services/config/i18next.js

```diff
@@ -23,8 +23,8 @@ i18next
   .use(initReactI18next)
   // for all options read: https://www.i18next.com/overview/configuration-options
   .init({
-    fallbackLng: ConfigService.getParameterValue('FALLBACK_LANGUAGE'),
-    supportedLngs: Object.keys(ConfigService.getParameterValue('LANGUAGES')),
+    fallbackLng: FALLBACK_LANGUAGE,
+    supportedLngs: LANGUAGES,
```

Comment:
`LANGUAGES` must be replaced by `Object.keys(LANGUAGES)` (the previous line of code was `supportedLngs: Object.keys(ConfigService.getParameterValue('LANGUAGES')),`)

Note that the unit tests are broken, it may be because of this bug